### DISCUSSION TITLE
ERROR: prepared statement "s0" does not exist

### DISCUSSION THREAD

**Discussion by thebadcivilian** [supabase/supabase#20015] [Questions]
Hello everyone. My name is Raiyan. I have installed ORM prisma@4.8.1 in nextJs and created a project on supabase but after adding model i am running npx prisma db push but instead of migrating i am getting error ---> Migration engine error:
db error: ERROR: prepared statement "s0" does not exist. Please help.

**Comment by GaryAustin1** [comment]
You should look at the connection strings carefully and make sure you are using the regular port and pooler ports for the right things.  Also  make sure you have all the extra flags in your pooler string.
See info here:  https://supabase.com/partners/integrations/prisma
